pom.xml

```diff
@@ -483,6 +483,22 @@ THE SOFTWARE.
           <rerunFailingTestsCount>4</rerunFailingTestsCount>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>run-javadoc</id>
```

Comment:
You could do this only in a profile if `!skipTests` to speed up developer builds, as we do in core & plugin builds.